### DISCUSSION TITLE
[FEATURE REQ] Max Download - per indexer

### DISCUSSION THREAD

**Discussion by Audionut** [autobrr/autobrr#1689] [Ideas]
**Is your feature request related to a problem? Please describe.**
First world problem.

**Describe the solution you'd like**
Max Download being specific to each tracker added to the filter. So if you have 1 download per day, it will download 1 thing from each tracker in the filter, each day, rather than only 1 thing each day, period.

**Describe alternatives you've considered**
None, I'm lazy.

**Comment by zze0s** [comment]
This would add quite a bit of complexity which I'm not too sure about.

The easy solution is to duplicate and split the filter per indexer to achieve a similar thing.
